src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -506,7 +509,7 @@ function WebRTCAdaptor(initialValues)
 
 		var jsCmd = {
 				command : "leave",
-				streamId: streamId,
+				streamId: thiz.streamId,
```

Comment:
streamId is already provided in the method. Why are using object's variable(thiz.streamId)?